uwblueprint/planet-read
# Postgres to MySQL migration

## Notion ticket link

[Swap Postgres for MySQL](https://www.notion.so/uwblueprintexecs/Swap-Postgres-for-MySQL-ac879226df3945989761eedd6df8125c)



## Implementation description
* Replaced the configuration for Postgres with MySQL
* Updated table schemas (primarily string and array cols) 
* Updated APIs 



## Steps to test
1. add and set these variables to your dot env:
```
MYSQL_DB=
MYSQL_USER=
MYSQL_PASSWORD=
MYSQL_HOST=
```
2. in `backed/python/app/models/__init__.py` set `erase_db_and_sync = True`  
3. make sure the DB is up and running, and all the tables are there
4. set erase_db_and_sync = False`
5. populate the database 
6. test the mutations and queries (I found it easy to test via http://localhost:5000/graphql)


## What should reviewers focus on?
* queries and mutations!


## Checklist
